docs/HowToConnectExternally.md

```diff
@@ -0,0 +1,299 @@
+# Connect to NuoDB Database Externally
+
+## Introduction
+
+NuoDB supports SQL clients via a NuoDB-provided client driver such as JDBC.
+NuoDB provides a configurable internal load balancer that can direct clients to Transaction Engines (TEs) according to user-specified load balancing rules.
+This enables the administrator to direct different client workloads to different pools of TEs in order to maximize cache locality and execute certain workloads on a TE with the best configuration for that workload.
+
+When a SQL client connects to a database, the driver first establishes a connection with the NuoDB Admin, which will perform load balancing and redirect the client to an appropriate TE in that database.
+Load balancing can be configured using the powerful `LBQuery` language to define the database connection strategy for routing SQL clients to the most appropriate TEs.
+
+For more information on NuoDB client connections, see [Client Development](https://doc.nuodb.com/nuodb/latest/client-development/).
+
+For more information on the LBQuery language syntax, see [Load Balancer Policies](https://doc.nuodb.com/nuodb/latest/client-development/load-balancer-policies/).
+
+A _direct_ connection to TEs is also supported by NuoDB, however, in this case, the load balancing must be performed by the client application.
+
+SQL clients and applications running in the same Kubernetes cluster as the NuoDB domain can connect to the database with the default NuoDB Helm Charts configuration using the NuoDB Admin `ClusterIP` service or directly using the TE `ClusterIP` service. This document focuses on external client applications running outside of the Kubernetes cluster where the NuoDB database is hosted. Allowing external access to the NuoDB database is not enabled by default and requires additional configuration.
+
+## Transaction Engine Groups
+
+A TE group consists of TEs with the same configuration which is part of the same database most often used to serve specific SQL workload.
+NuoDB Helm Charts 3.4.0+ supports the deployment of one or more TE groups per database.
+
+Multiple deployments of the `database` chart for the same NuoDB database can be done in the same Kubernetes namespace where only one of them is _primary_.
+One or more _secondary_ Helm releases are used to deploy additional TE groups for the same database with different configuration options.
+This allows flexible configuration of each TE group including but not limited to the number of TEs in a group, their resource requirements, process labels, and scheduling rules.
```

Comment:
Can we reword this to make it clearer that TEs within the same group have identical process labels (used by LBQuery) and K8s scheduling rules, and can be scaled independently of each other?